src/panels/lovelace/cards/types.ts

```diff
@@ -275,4 +275,5 @@ export interface WeatherForecastCardConfig extends LovelaceCardConfig {
   entity: string;
   name?: string;
   show_forecast?: boolean;
+  attribute?: string;
```

Comment:
Should we name this `secondary_info`?

Reply:
Or `secondary_info_attribute`?

Reply:
Yea I had it as secondary attribute first but saw we used attribute other places. Didn't know if this should follow that. I like `secondary_info_attribute`

Reply:
Where do we use `attribute`? Is it when it is a primary thing (like entity card) or for secondary too?

Reply:
I thought the entities card did. But it is also for primary